jumeg/connectivity/con_viz.py

```diff
@@ -390,7 +390,7 @@ def plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
     end_noise = rng.uniform(-noise_max, noise_max, n_con)
 
     nodes_n_con_seen = np.zeros_like(nodes_n_con)
-    for i, (start, end) in enumerate(zip(indices[0], indices[1])):
+    for i, (end, start) in enumerate(zip(indices[0], indices[1])):
```

Comment:
I am wondering if it is better to transpose the matrix when arrow=True (when we know that it is causal matrix) instead of making the change here. That way the plotting code is retained similar to that of the original plot_connectivity_circle. What do you think?

Reply:
For the section you highlighted it does not really matter, since it is just the noise added to avoid that the arrows overlap in the plot. I could as well just have left it the way it was. The second section where I swapped i and j is the important part.

Anyways, I would rather not transpose the causality matrix because causality is defined as the columns driving the rows (j -> i) and now the code reflects that, explicitly stating that the arrows start at j and end at i. I fear it might be confusing otherwise.

Reply:
Alright, then you can leave it as it is - i.e. with changes both to the noise part and the main arrow drawing part so that it is uniform throughout.